### PR TITLE
Organization endpoints

### PR DESCRIPTION
**Pull request by ackers93**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (keveightysev) [inline .vscode/settings.json]: add the .vscode/ folder to gitignore and delete it from the repo please

File: api/employee/employeeModel.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const db = require('../../data/dbConfig');
```
Comment (keveightysev) [inline api/employee/employeeModel.js]: I think we'll want a model that gets all employees by org id

File: api/employee/employeeRouter.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
+const router = require('express').Router();
+const emp = require('./employeeModel.js');
+const userModel = require('../user/userModel.js');
+
+const auth = require('../../middleware/authMiddleWare');
+
+router.use(auth.validateId);
+
+// Get all employees from the database
+router.get('/', (req, res) => {
```
Comment (keveightysev) [inline api/employee/employeeRouter.js]: we could get the org id from req.profile and find all employees for a single organization

File: api/user/userRouter.js [hunk 4]
```diff
@@ -36,6 +36,7 @@ router.post('/', (req, res) => {
 	if (user.picture) {
 		body.profile_picture = user.picture;
 	}
+	console.log(body, 'body');
```
Comment (keveightysev) [inline api/user/userRouter.js]: please remove console logs

File: middleware/authMiddleWare.js [hunk 5]
```diff
@@ -39,6 +39,7 @@ module.exports.validateId = (req, res, next) => {
 				res.status(200).json({ user: false });
 			} else {
 				req.profile = user;
+				console.log(req.profile, 'req.profile');
```
Comment (keveightysev) [inline middleware/authMiddleWare.js]: please remove console logs

File: .vscode/settings.json (hunk 1 above)
Comment (Hail91) [reply .vscode/settings.json]: All set.

File: api/employee/employeeModel.js (hunk 2 above)
Comment (Hail91) [reply api/employee/employeeModel.js]: This is all set.

File: api/employee/employeeRouter.js (hunk 3 above)
Comment (Hail91) [reply api/employee/employeeRouter.js]: Done.

File: api/user/userRouter.js (hunk 4 above)
Comment (Hail91) [reply api/user/userRouter.js]: All set!

File: middleware/authMiddleWare.js (hunk 5 above)
Comment (Hail91) [reply middleware/authMiddleWare.js]: All set!

**Pull request merged**